data_driven_acquisition/templates/base_with_sidebar.html

```diff
@@ -10,11 +10,16 @@
         <div class="grid-row grid-gap">
             <aside class="usa-layout-docs__sidenav desktop:grid-col-3">
                 <h1 class="text-primary">{{package.name}}</h1>
-                <nav aria-label="Secondary navigation">
+                <nav aria-label="Secondary navigation" class="position-sticky top-2">
                     <ul class="usa-sidenav usa-accordion dda-sidenav-tree" aria-multiselectable="true">
                         <li class="usa-sidenav__item">
-                            {% url 'package' package_id=package.id as link %}
-                            <a href="{{ link }}" class="{% if link == request.path %}usa-current{% endif %}">Acquisition properties</a>
+                            {% block sidenav_first_item %}
+                                {% url 'package' package_id=package.id as link %}
+                                <a href="{{ link }}" class="{% if link == request.path %}usa-current{% endif %}">
+                                    <i class="fas fa-file-alt margin-right-05"></i>
+                                    Acquisition properties
+                                </a>
+                            {% endblock %}
```

Comment:
These changes added in #22 to allow the acquisition property editors to add items (the form groups) to the sidebar.